### PR TITLE
adds version and help short hand arguments

### PR DESCRIPTION
**Pull request by ykethan**
## Problem

**Issue number, if available:**

## Changes

adds `-v` and `-h` alias for version and help commands

**Corresponding docs PR, if applicable:**

## Checklist

- [ ] If this PR includes a functional change to the runtime behavior of the code, I have added or updated automated test coverage for this change.
- [ ] If this PR requires a change to the [Project Architecture README](../PROJECT_ARCHITECTURE.md), I have included that update in this PR.
- [ ] If this PR requires a docs update, I have linked to that docs PR above.
- [ ] If this PR modifies E2E tests, makes changes to resource provisioning, or makes SDK calls, I have run the PR checks with the `run-e2e` label set.

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

File: packages/cli/src/main_parser_factory.ts [hunk 1]
```diff
@@ -29,6 +30,7 @@ export const createMainParser = (libraryVersion: string): Argv => {
     .command(createConfigureCommand())
     .command(createInfoCommand())
     .help()
+    .alias('h', 'help')
```
Comment (sobolk) [inline packages/cli/src/main_parser_factory.ts:33]: could you add test coverage for the alias here https://github.com/aws-amplify/amplify-backend/blob/main/packages/cli/src/main_parser_factory.test.ts ?

File: packages/cli/src/main_parser_factory.ts [hunk 2]
```diff
@@ -12,6 +12,7 @@ import * as path from 'path';
 export const createMainParser = (libraryVersion: string): Argv => {
   const parser = yargs()
     .version(libraryVersion)
+    .alias('v', 'version')
```
Comment (sobolk) [inline packages/cli/src/main_parser_factory.ts:15]: nit: mind grouping `.alias` calls together ?

File: packages/cli/src/main_parser_factory.ts (hunk 1 above)
Comment (ykethan) [reply packages/cli/src/main_parser_factory.ts:32]: added the tests for the shorthand versions

File: packages/cli/src/main_parser_factory.ts (hunk 2 above)
Comment (ykethan) [reply packages/cli/src/main_parser_factory.ts]: grouped the alias calls

**Review by sobolk — APPROVED** [review]

**Review by rtpascual — APPROVED** [review]

**Pull request merged**